### ISSUE TITLE
`block_201` raises an error on comment lines when `is` is missing

### ISSUE THREAD

**Issue by JHertz5** [jeremiah-c-leary/vhdl-style-guide#1564]
**Environment**
git hash a895557eee9ce0fb9355828758f217c11563aa47 (HEAD of the master branch at time of writing)

**Describe the bug**
`block_201` is flagging errors on code that isn't even within a block statement when the `is` of the block statement (which is optional) is missing and the file starts with comments. If the `is` is present, or if the comments are removed, the warning goes away. If `--fix` is used, VSG "fixes" the issue by putting a blank line between the comment lines at the top.

**To Reproduce**
1. RTL file: test.vhd
```vhdl
--
--

architecture rtl of test is
begin
  my_label : block

  begin
  end block;
end architecture;
```
2. Config file: test.yml
```yaml
rule:
    global:
        disable: true
    block_201:
        disable: false
```
3. Run `$ vsg -f test.vhd -c test.yml`
```
================================================================================
File:  test.vhd
================================================================================
Phase 3 of 7... Reporting
Total Rules Checked: 1
Total Violations:    1
  Error   :     1
  Warning :     0
------------+------------+------------+--------------------------------------
  Rule      |  severity  |  line(s)   | Solution
------------+------------+------------+--------------------------------------
  block_201 | Error      |          1 | Insert blank line below
------------+------------+------------+--------------------------------------
NOTE: Refer to online documentation at https://vhdl-style-guide.readthedocs.io/en/latest/index.html for more information.
```

**Expected behavior**
There should be no error in this case, `block_201` should be satisfied by this code.
